### ISSUE TITLE
Empty ULs in the HEADER

### ISSUE THREAD

**Issue by timo54** [ATLAS-Illinois/Illinois-Framework-Theme#1086]
I’ve encountered empty UL’s on the Biology page ([biology.illinois.edu](https://biology.illinois.edu/)). At the top of each page on the site, as part of the Illinois Framework Project heading - ilw-header, there is an empty UL with aria-label=”Utility”. The empty UL is flagged as a violation by AInspector because each list is required to have a listitem.

**Comment by trubach** [comment]
This is an issue in the twig [region--header.html.twig](https://github.com/web-illinois/illinois_framework_theme/blob/d70cc79420244b240c1bd5d0cbb560dfa1146c72/templates/region/region--header.html.twig#L41-L51). It assumes there will be links in the area above the search bar. We should add a check to the twig template so that the `<nav>` and `<ul>` elements don't render if the links are empty.

This will need to be addressed in the `4.x` and `5.x` branches

- [x] Fix issue in 4.x
- [ ] Fix issue in 5.x

**Comment by tjackson2113** [comment]
5.x did not refactor correctly. 4.x is fine.
